lib/analytics.js

```diff
@@ -45,6 +46,7 @@ var type = require('component-type');
 function Analytics() {
   this._options({});
   this.Integrations = {};
+  this.IntegrationMiddlewares = new MiddlewareChain();
```

Comment:
can we name this `this_integrationMiddlewares`?

Reply:
Sure, that's definitely reasonable!